src/main.rs

```diff
@@ -54,6 +54,7 @@ mod console_log;
 mod deps;
 mod fork;
 mod formatter;
+mod http_fork_source;
```

Comment:
Unsure where else to put this comment, but please include a screenshot of `era_test_node fork testnet` working locally for you as Evidence in the PR description

Reply:
done.